test.js

```diff
@@ -1,189 +1,219 @@
 #!/usr/bin/env node
 const path = require("path");
-require('dotenv').config();
+require("dotenv").config();
 const { default: computeTVL } = require("@defillama/sdk/build/computeTVL");
 const { chainsForBlocks } = require("@defillama/sdk/build/computeTVL/blocks");
 const { getLatestBlock } = require("@defillama/sdk/build/util/index");
-const { humanizeNumber } = require("@defillama/sdk/build/computeTVL/humanizeNumber");
+const {
+  humanizeNumber,
+} = require("@defillama/sdk/build/computeTVL/humanizeNumber");
 const { util } = require("@defillama/sdk");
 
 async function getLatestBlockRetry(chain) {
-    for (let i = 0; i < 5; i++) {
-        try {
-            return await getLatestBlock(chain);
-        } catch (e) {
-            throw new Error(`Couln't get block heights for chain "${chain}"`, e)
-        }
-    }
+  // for (let i = 0; i < 5; i++) {
```

Comment:
why use a fixed block?

Reply:
When I try to run test.js, I got error "Couln't get block heights for chain"
so I have used fixed block for testing

Reply:
I have reverted test.js code